docs/reference/commandline/run.md

```diff
@@ -539,3 +540,31 @@ the three processes quota set for the `daemon` user.
 The `--stop-signal` flag sets the system call signal that will be sent to the container to exit.
 This signal can be a valid unsigned number that matches a position in the kernel's syscall table, for instance 9,
 or a signal name in the format SIGNAME, for instance SIGKILL.
+
+### Specify isolation technology for container (--isolation)
```

Comment:
Thanks! I see some issues in the text here, I'll ask @moxiegirl if she can have a look (she's usually faster and better at that than I am)